GiniVision/Classes/Core/ImageMetaInformationManager.swift

```diff
@@ -266,7 +266,7 @@ internal class ImageMetaInformationManager {
         return existingUUID ?? NSUUID().uuidString
     }
     
-    fileprivate func imageRotationDeltaDegrees() -> Int {
+    func imageRotationDeltaDegrees() -> Int {
```

Comment:
Why would an external class need to know the rotation?

Reply:
The `GiniImageDocument` class needs to know the rotation angle in order to be used by other classes.